crates/sui-source-validation-service/src/lib.rs

```diff
@@ -59,12 +59,12 @@ pub fn host_port() -> String {
 
 #[derive(Deserialize, Debug)]
 pub struct Config {
-    pub packages: Vec<PackageSources>,
+    pub packages: Vec<PackageSource>,
 }
 
 #[derive(Deserialize, Debug)]
 #[serde(tag = "source", content = "values")]
-pub enum PackageSources {
+pub enum PackageSource {
```

Comment:
main change: this type now represents a single `PackageSource`, so I'm just removing the plural